src/Symfony/Bridge/Doctrine/Tests/Form/Type/EntityTypeTest.php

```diff
@@ -1771,4 +1771,40 @@ public function testWithSameLoaderAndDifferentChoiceValueCallbacks()
         $this->assertSame('Foo', $view['entity_two']->vars['choices']['Foo']->value);
         $this->assertSame('Bar', $view['entity_two']->vars['choices']['Bar']->value);
     }
+
+    public function testFieldMappingAsObject()
+    {
+        $deprecation = [];
+
+        set_error_handler(static function (int $errno, string $errstr) use (&$deprecation) {
+            $deprecation[] = $errstr;
+        }, E_USER_DEPRECATED);
```

Comment:
We should not need a dedicated error handler for deprecations. Our error handler should make our tests fail if they trigger a deprecation. In other words: test the functionality, not the deprecation.

Reply:
so delete the test, the test for this fonctionnality already exists,

Reply:
I delete it